wpeverest/everest-forms
# Fix - Form toggle button

### All Submissions:






### Changes proposed in this Pull Request:

This PR solves the issue of the Form status Toggle button not working issue.

Closes https://wordpress.org/support/topic/enable-disable-form/

### How to test the changes in this Pull Request:

1. Try to Toggle the form where the forms are listed.
2. Reload the page and see the forms are being Toggled or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form status {de}active toggle button.
